shared/src/test/diff/nu/Refinements.mls

```diff
@@ -85,15 +85,17 @@ let r = new {}
 //│ Code generation encountered an error:
 //│   Unsupported `new` class term: Bra(rcd = true, Rcd())
 
+:ge
 r : Object
 //│ Object
-//│ res
-//│     = {}
+//│ Code generation encountered an error:
```

Comment:
Actually is this happening because some mutable state is not reset when the previous code-gen error is thrown? This could be fixed easily by using a `try`/`finally`. If not and it's harder to fix, pls disregard my question and let's merge.

Reply:
I tried and found it hard to fix for now: we added symbols into the scope for top-level mutually-recursion and it is hard to undo it (the previous symbol in `lexicalValueSymbols` is gone). I will trace this in issue #65 and fix it when we refactor the `JSBackend`.